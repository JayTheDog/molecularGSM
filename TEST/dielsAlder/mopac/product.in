PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization
  C -0.055074 -1.424969 0.379886
  C -1.219569 -0.723523 -0.272991
  C -1.280757 0.610521 -0.336412
  C -0.121075 1.444865 0.152554
  C 1.311169 -0.685702 0.224614
  C 1.197474 0.772102 -0.273551
  H -2.039097 -1.334283 -0.648784
  H -0.276628 -1.520162 1.451456
  H 0.025745 -2.451913 0.011092
  H 1.830654 -0.705537 1.189197
  H 1.945311 -1.242478 -0.473524
  H 1.228708 0.785455 -1.369785
  H 2.060455 1.353185 0.074529
  H -0.174779 2.463364 -0.249464
  H -0.154258 1.543788 1.249963
  H -2.134925 1.106333 -0.788776
