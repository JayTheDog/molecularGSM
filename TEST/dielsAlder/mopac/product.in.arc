

                     SUMMARY OF PM6 CALCULATION, Site No: 14762

                                                       MOPAC2012 (Version: 15.168L)
                                                       Fri Dec  2 19:40:46 2016
                                                       No. of days left =   0

           Empirical Formula: C6 H10  =    16 atoms

PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =         -4.01596 KCAL/MOL =     -16.80277 KJ/MOL
          TOTAL ENERGY            =       -870.90978 EV
          ELECTRONIC ENERGY       =      -3985.57034 EV
          CORE-CORE REPULSION     =       3114.66056 EV
          GRADIENT NORM           =          0.90640
          DIPOLE                  =          0.36830 DEBYE    POINT GROUP:       C1  
          NO. OF FILLED LEVELS    =         17
          IONIZATION POTENTIAL    =          9.405656 EV
          HOMO LUMO ENERGIES (EV) =         -9.406  1.517
          MOLECULAR WEIGHT        =         82.145
          COSMO AREA              =        129.85 SQUARE ANGSTROMS
          COSMO VOLUME            =        121.83 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H    16    H    11     5.00361
            H    13    H     8     4.32207
            H    10    H     9     2.75968
          SCF CALCULATIONS        =         53
          WALL-CLOCK TIME         =          0.312 SECONDS
          COMPUTATION TIME        =          0.309 SECONDS


          FINAL GEOMETRY OBTAINED
PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization
  C     0.02034153 +1  -1.49621837 +1   0.05864551 +1
  C    -1.25278056 +1  -0.71194644 +1  -0.07243270 +1
  C    -1.29352845 +1   0.62275484 +1  -0.13530014 +1
  C    -0.07082808 +1   1.48910984 +1  -0.05302902 +1
  C     1.21436261 +1  -0.62121074 +1   0.47379000 +1
  C     1.22214892 +1   0.69848730 +1  -0.30912896 +1
  H    -2.15864900 +1  -1.31348450 +1  -0.12854798 +1
  H    -0.11495076 +1  -2.31620308 +1   0.79188622 +1
  H     0.23560774 +1  -1.98961141 +1  -0.91300942 +1
  H     1.16015343 +1  -0.40673351 +1   1.55837846 +1
  H     2.15998340 +1  -1.16925223 +1   0.30981533 +1
  H     1.31815425 +1   0.48624977 +1  -1.39127397 +1
  H     2.10202651 +1   1.30557216 +1  -0.03067043 +1
  H    -0.14842658 +1   2.32585509 +1  -0.77454717 +1
  H    -0.03545451 +1   1.95534102 +1   0.95440379 +1
  H    -2.23331467 +1   1.16253992 +1  -0.23584899 +1
 
