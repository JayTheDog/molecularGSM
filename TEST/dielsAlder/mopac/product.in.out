 *******************************************************************************
 ** Site#: 14762        For non-commercial use only     Version 15.168L 64BITS**
 *******************************************************************************
 ** Cite this work as:                                                        **
 **                                                                           **
 ** (a) MOPAC2012, James J. P. Stewart, Stewart Computational Chemistry,      **
 **  Version 15.168L web: HTTP://OpenMOPAC.net         Days remaining:   0    **
 **                                                                           **
 **                                 AND                                       **
 **                                                                           **
 ** (b) Maia, J.D.C., et al., J. Chem. Theory Comput. 2012, 8, 3072-3081.     **
 **                DOI: 10.1021/ct3004645.                                    **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2012                                  **
 **                                                                           **
 *******************************************************************************

                              PM6 CALCULATION RESULTS


 *******************************************************************************
 *  CALCULATION DONE:                                Fri Dec  2 19:40:45 2016  *
 *  PM6        - The PM6 Hamiltonian to be used
 *  SINGLET    - SPIN STATE DEFINED AS A SINGLET
 *  NOSYM      - POINT-GROUP SYMMETRY SET TO C1
 *  T=         - A TIME OF     10.0 MINUTES REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C         -0.05507400  *  -1.42496900  *   0.37988600  *
     2       C         -1.21956900  *  -0.72352300  *  -0.27299100  *
     3       C         -1.28075700  *   0.61052100  *  -0.33641200  *
     4       C         -0.12107500  *   1.44486500  *   0.15255400  *
     5       C          1.31116900  *  -0.68570200  *   0.22461400  *
     6       C          1.19747400  *   0.77210200  *  -0.27355100  *
     7       H         -2.03909700  *  -1.33428300  *  -0.64878400  *
     8       H         -0.27662800  *  -1.52016200  *   1.45145600  *
     9       H          0.02574500  *  -2.45191300  *   0.01109200  *
    10       H          1.83065400  *  -0.70553700  *   1.18919700  *
    11       H          1.94531100  *  -1.24247800  *  -0.47352400  *
    12       H          1.22870800  *   0.78545500  *  -1.36978500  *
    13       H          2.06045500  *   1.35318500  *   0.07452900  *
    14       H         -0.17477900  *   2.46336400  *  -0.24946400  *
    15       H         -0.15425800  *   1.54378800  *   1.24996300  *
    16       H         -2.13492500  *   1.10633300  *  -0.78877600  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         C       -0.0551   -1.4250    0.3799
     2         C       -1.2196   -0.7235   -0.2730
     3         C       -1.2808    0.6105   -0.3364
     4         C       -0.1211    1.4449    0.1526
     5         C        1.3112   -0.6857    0.2246
     6         C        1.1975    0.7721   -0.2736
     7         H       -2.0391   -1.3343   -0.6488
     8         H       -0.2766   -1.5202    1.4515
     9         H        0.0257   -2.4519    0.0111
    10         H        1.8307   -0.7055    1.1892
    11         H        1.9453   -1.2425   -0.4735
    12         H        1.2287    0.7855   -1.3698
    13         H        2.0605    1.3532    0.0745
    14         H       -0.1748    2.4634   -0.2495
    15         H       -0.1543    1.5438    1.2500
    16         H       -2.1349    1.1063   -0.7888

 General Reference for PM6:
 "Optimization of Parameters for Semiempirical Methods V: Modification of NDDO Approximations
 and Application to 70 Elements", J. J. P. Stewart, J. Mol. Mod., 13, 1173-1213 (2007)
 URL: http://www.springerlink.com/content/ar33482301010477/fulltext.pdf

           Empirical Formula: C6 H10  =    16 atoms



      MOLECULAR POINT GROUP   :   C1  


 SINGLET STATE CALCULATION


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =   17

      Geometry optimization using EF

          DIAGONAL MATRIX USED AS START HESSIAN

 CYCLE:     1 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:    33.403 HEAT:-0.6340264
 CYCLE:     2 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:    36.540 HEAT:-0.7211163
 CYCLE:     3 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:    37.781 HEAT:-0.8285707
 CYCLE:     4 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:    19.210 HEAT: -1.211421
 CYCLE:     5 TIME:   0.016 TIME LEFT: 10.00M  GRAD.:    19.790 HEAT: -1.270915
 CYCLE:     6 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:    13.027 HEAT: -1.334556
 CYCLE:     7 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:    12.532 HEAT: -1.372397
 CYCLE:     8 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:    12.548 HEAT: -1.385672
 CYCLE:     9 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:    11.965 HEAT: -1.405620
 CYCLE:    10 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:    12.878 HEAT: -1.406637
 CYCLE:    11 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     4.922 HEAT: -1.458665
 CYCLE:    12 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     2.522 HEAT: -1.467188
 CYCLE:    13 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     2.859 HEAT: -1.471249
 CYCLE:    14 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     2.802 HEAT: -1.476762
 CYCLE:    15 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     3.203 HEAT: -1.483374
 CYCLE:    16 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     2.510 HEAT: -1.492917
 CYCLE:    17 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     3.521 HEAT: -1.506213
 CYCLE:    18 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     4.543 HEAT: -1.520071
 CYCLE:    19 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     3.676 HEAT: -1.536714
 CYCLE:    20 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     3.417 HEAT: -1.557285
 CYCLE:    21 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     3.672 HEAT: -1.578361
 CYCLE:    22 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     4.419 HEAT: -1.604176
 CYCLE:    23 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     4.416 HEAT: -1.637355
 CYCLE:    24 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     4.992 HEAT: -1.675416
 CYCLE:    25 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     5.633 HEAT: -1.724650
 CYCLE:    26 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     6.007 HEAT: -1.781413
 CYCLE:    27 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     9.000 HEAT: -1.842262
 CYCLE:    28 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     9.296 HEAT: -1.964088
 CYCLE:    29 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:    11.537 HEAT: -2.142515
 CYCLE:    30 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:    17.076 HEAT: -2.481279
 CYCLE:    31 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:    16.059 HEAT: -2.692976
 CYCLE:    32 TIME:   0.016 TIME LEFT: 10.00M  GRAD.:    15.000 HEAT: -2.802663
 CYCLE:    33 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     7.140 HEAT: -3.041041
 CYCLE:    34 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     6.736 HEAT: -3.208435
 CYCLE:    35 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     8.905 HEAT: -3.417822
 CYCLE:    36 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:    11.677 HEAT: -3.656490
 CYCLE:    37 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     7.698 HEAT: -3.849304
 CYCLE:    38 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     5.341 HEAT: -3.923247
 CYCLE:    39 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     4.360 HEAT: -3.955914
 CYCLE:    40 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     4.650 HEAT: -3.967167
 CYCLE:    41 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     3.061 HEAT: -3.989493
 CYCLE:    42 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     2.139 HEAT: -3.999016
 CYCLE:    43 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     1.404 HEAT: -4.007067
 CYCLE:    44 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     1.280 HEAT: -4.011289
 CYCLE:    45 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     1.062 HEAT: -4.013824
 CYCLE:    46 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     0.906 HEAT: -4.015959

     GRADIENT =  0.90640  IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   


                              PM6 CALCULATION
                                                       MOPAC2012 (Version: 15.168L)
                                                       Fri Dec  2 19:40:46 2016
                                                       No. of days left =   0




          FINAL HEAT OF FORMATION =         -4.01596 KCAL/MOL =     -16.80277 KJ/MOL


          TOTAL ENERGY            =       -870.90978 EV
          ELECTRONIC ENERGY       =      -3985.57034 EV  POINT GROUP:     C1  
          CORE-CORE REPULSION     =       3114.66056 EV
          COSMO AREA              =        129.85 SQUARE ANGSTROMS
          COSMO VOLUME            =        121.83 CUBIC ANGSTROMS

          GRADIENT NORM           =          0.90640
          IONIZATION POTENTIAL    =          9.405656 EV
          HOMO LUMO ENERGIES (EV) =         -9.406  1.517
          NO. OF FILLED LEVELS    =         17
          MOLECULAR WEIGHT        =         82.145

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H    16    H    11     5.00361
            H    13    H     8     4.32207
            H    10    H     9     2.75968


          SCF CALCULATIONS        =         53
 
          WALL-CLOCK TIME         =          0.312 SECONDS
          COMPUTATION TIME        =          0.308 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C          0.02034153  *  -1.49621837  *   0.05864551  *
     2       C         -1.25278056  *  -0.71194644  *  -0.07243270  *
     3       C         -1.29352845  *   0.62275484  *  -0.13530014  *
     4       C         -0.07082808  *   1.48910984  *  -0.05302902  *
     5       C          1.21436261  *  -0.62121074  *   0.47379000  *
     6       C          1.22214892  *   0.69848730  *  -0.30912896  *
     7       H         -2.15864900  *  -1.31348450  *  -0.12854798  *
     8       H         -0.11495076  *  -2.31620308  *   0.79188622  *
     9       H          0.23560774  *  -1.98961141  *  -0.91300942  *
    10       H          1.16015343  *  -0.40673351  *   1.55837846  *
    11       H          2.15998340  *  -1.16925223  *   0.30981533  *
    12       H          1.31815425  *   0.48624977  *  -1.39127397  *
    13       H          2.10202651  *   1.30557216  *  -0.03067043  *
    14       H         -0.14842658  *   2.32585509  *  -0.77454717  *
    15       H         -0.03545451  *   1.95534102  *   0.95440379  *
    16       H         -2.23331467  *   1.16253992  *  -0.23584899  *

                             CARTESIAN COORDINATES

   1    C        0.020341526    -1.496218373     0.058645511
   2    C       -1.252780557    -0.711946442    -0.072432699
   3    C       -1.293528451     0.622754843    -0.135300138
   4    C       -0.070828081     1.489109844    -0.053029021
   5    C        1.214362608    -0.621210736     0.473790004
   6    C        1.222148922     0.698487297    -0.309128963
   7    H       -2.158648998    -1.313484501    -0.128547977
   8    H       -0.114950764    -2.316203080     0.791886225
   9    H        0.235607739    -1.989611408    -0.913009420
  10    H        1.160153427    -0.406733507     1.558378455
  11    H        2.159983399    -1.169252234     0.309815327
  12    H        1.318154252     0.486249766    -1.391273973
  13    H        2.102026514     1.305572159    -0.030670432
  14    H       -0.148426579     2.325855087    -0.774547169
  15    H       -0.035454506     1.955341019     0.954403786
  16    H       -2.233314668     1.162539924    -0.235848986


           Empirical Formula: C6 H10  =    16 atoms



      MOLECULAR POINT GROUP   :   C1  


                  EIGENVALUES  
 -29.24042 -25.83833 -25.67575 -21.48192 -20.82814 -17.52389 -16.70784 -15.03507
 -14.38390 -13.82403 -13.24215 -13.01507 -12.86249 -11.38650 -11.21097 -10.91785
  -9.40566   1.51674   4.12828   4.18265   4.61097   4.72599   4.96563   5.69173
   5.80793   5.94872   6.10014   6.21418   6.36488   6.46411   6.51358   6.57662
   6.64271   6.71608


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          C          -0.256116        4.2561     1.08188     3.17424
    2          C          -0.156601        4.1566     1.10977     3.04683
    3          C          -0.156994        4.1570     1.10975     3.04724
    4          C          -0.256273        4.2563     1.08178     3.17449
    5          C          -0.245464        4.2455     1.08564     3.15982
    6          C          -0.245347        4.2453     1.08547     3.15988
    7          H           0.131987        0.8680     0.86801
    8          H           0.132731        0.8673     0.86727
    9          H           0.138839        0.8612     0.86116
   10          H           0.132369        0.8676     0.86763
   11          H           0.122129        0.8779     0.87787
   12          H           0.132791        0.8672     0.86721
   13          H           0.122112        0.8779     0.87789
   14          H           0.132383        0.8676     0.86762
   15          H           0.139332        0.8607     0.86067
   16          H           0.132122        0.8679     0.86788
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.366     0.003     0.034     0.368
 HYBRID         0.000    -0.002     0.001     0.002
 SUM            0.367     0.001     0.034     0.368


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  C   1.08188   0.99335   1.05006   1.13083
    2  C   1.10977   1.02662   1.00257   1.01764
    3  C   1.10975   1.03122   0.99780   1.01822
    4  C   1.08178   0.98821   1.04996   1.13631
    5  C   1.08564   1.04713   1.01284   1.09985
    6  C   1.08547   1.03344   1.01833   1.10811
    7  H   0.86801
    8  H   0.86727
    9  H   0.86116
   10  H   0.86763
   11  H   0.87787
   12  H   0.86721
   13  H   0.87789
   14  H   0.86762
   15  H   0.86067
   16  H   0.86788

          JOB ENDED NORMALLY



 TOTAL JOB TIME:             0.31 SECONDS

 == MOPAC DONE ==
